src/Altinn.Platform/Altinn.Platform.Storage/IntegrationTest/InstanceMultipartTests.cs

```diff
@@ -0,0 +1,180 @@
+            }
+            catch (Exception)
+            {
+                // do nothing.
+            }
+
+            LanguageString title = new LanguageString
+            {
+                { "nb", "Testapplikasjon" },
+                { "en", "Test application" }
+            };
+
+            return appClient.CreateApplication(testAppId, title);
+        }
+
+        private Application DeleteApplicationMetadata()
+        {
+            ApplicationClient appClient = new ApplicationClient(client);
+
+            Application existingApp = appClient.DeleteApplication(testAppId);
+
+            return existingApp;
+        }
+
+        /// <summary>
+        /// Store a multipart file in one post operation
+        /// </summary>
+        [Fact]
+        public async void StoreMultiPartFileInOnePostOperation()
+        {
+            Instance instance = new Instance()
+            {
+                Id = Guid.NewGuid().ToString(),
+                InstanceOwnerId = "1000",
+                AppId = "tests/sailor",
+                Labels = new List<string>()
+                {
+                    "Hei"
+                },
+                DueDateTime = DateTime.Parse("2019-10-01")
+            };
+            
+            MultipartFormDataContent form = new MultipartFormDataContent();
+
+            form.Add(instance.AsJson(), "instance");
+            string xmlText = File.ReadAllText("data/example.xml");
+
+            form.Add(new StringContent(xmlText, Encoding.UTF8, "application/xml"), "default");
+        
+            string requestUri = $"{versionPrefix}/instances?appId={instance.AppId}&instanceOwnerId={instance.InstanceOwnerId}";
+
+            HttpResponseMessage response = await client.PostAsync(requestUri, form);
+            
+            response.EnsureSuccessStatusCode();
+
+            string result = await response.Content.ReadAsStringAsync();
+
+            Instance instanceResult = JsonConvert.DeserializeObject<Instance>(result);
+
+            // Assert
+            Assert.Equal("1000", instanceResult.InstanceOwnerId);
+        }
+
+        /// <summary>
+        /// Store a json file in a post operation where it checks multipart.
+        /// </summary>
+        [Fact]
+        public async void StoreJsonInOnePostOperation()
+        {
+            Instance instance = new Instance()
+            {
+                Id = Guid.NewGuid().ToString(),
+                InstanceOwnerId = "1000",
+                AppId = "tests/sailor",
+                Labels = new List<string>()
+                {
+                    "Hei"
+                },
+                DueDateTime = DateTime.Parse("2019-10-01")
+            };
+
+            string requestUri = $"{versionPrefix}/instances?appId={instance.AppId}&instanceOwnerId={instance.InstanceOwnerId}";
+            
+            HttpResponseMessage postResponse = await client.PostAsync(requestUri, instance.AsJson());
+
+            postResponse.EnsureSuccessStatusCode();
```

Comment:
Here you should check that no data element is present.

Reply:
Yes, fixed.